test/libsolidity/ASTJSON/assembly/nested_functions.json

```diff
@@ -95,7 +95,6 @@
               }
             ]
           },
-          "functionSelector": "26121ff0",
```

Comment:
Is this change correct?

Reply:
Yes. The test itself contains invalid code which results in the AST not having type information for the `m_returnParameterTypes` which is what a new check in this PR looks for (among other things) to avoid null dereferencing.

We also shortly talked about this in the chat: https://gitter.im/ethereum/solidity-dev?at=5f46749489cf2d584b794faf

Reply:
Does this mean the selector here was also completely wrong?

Reply:
No, I don't think so. I just checked and the value here does not depend on the returnParameters, so it was unaffected by that.

Reply:
So it is available in some cases but the checks that were newly introduced are "too" strict to avoid exceptions being thrown in some other cases?

Reply:
Yes, that is an accurate summary. I am not sure it's an issue as the code is invalid anyway and wouldn't compile at all if we weren't exporting the AST.